backend/server.js

```diff
@@ -1,583 +1,664 @@
-require('dotenv').config();
+require("dotenv").config();
 
-const express = require('express');
-const http = require('http');
-const fs = require('fs');
+const express = require("express");
+const http = require("http");
+const fs = require("fs");
 
 const webServer = http.createServer((req, res) => {
-    if (req.url.includes('main.js')) {
-        res.writeHead(200, {'content-type': 'text/javascript'})
-        fs.createReadStream('frontend/main.js').pipe(res);
-    } else if (req.url.includes('styles.css')) {
-        res.writeHead(200, {'content-type': 'text/css'})
-        fs.createReadStream('frontend/styles.css').pipe(res);
-    } else if (req.url.includes('assets/')) {
-        const assetName = req.url.split('/').pop();
-        res.writeHead(200, {'content-type': 'image/png'})
-        fs.createReadStream(`frontend/assets/${assetName}`).pipe(res);
-    } else {
-        res.writeHead(200, {'content-type': 'text/html'})
-        fs.createReadStream('frontend/index.html').pipe(res);
-    }
+	if (req.url.includes("main.js")) {
```

Comment:
all of these changes in quotation marks and indentation seems a bit arbitrary to me, I'd accept them if they would be ensured by a linter, so there will be no changes in each PR, you know, depending on developer's IDE config

btw now the code is less readable to me, so from my perspective it would be best to leave old formatting and add linter config which would ensure it